src/diffusers/pipeline_utils.py

```diff
@@ -129,10 +129,14 @@ class DiffusionPipeline(ConfigMixin):
 
     Class attributes:
 
-        - **config_name** ([`str`]) -- name of the config file that will store the class and module names of all
+        - **config_name** (`str`) -- name of the config file that will store the class and module names of all
           components of the diffusion pipeline.
+        - **_optional_components** (List[`str`]) -- lists of all components that are optional so which don't have to be
+          passed for the pipeline to function (should be overridden by parent class).
     """
     config_name = "model_index.json"
+    _optional_components = []
+    _pipeline_kwargs = []
```

Comment:
What is this used for?

Reply:
True this is not used and can be removed